target/linux/ramips/image/rt305x.mk

```diff
@@ -250,6 +253,21 @@ define Device/dir-320-b1
 endef
 TARGET_DEVICES += dir-320-b1
 
+define Device/dlink_dir-506l
+  DTS := DIR-506L
```

Comment:
@rozhuk-im Please take a closer look at the `dwr-512-b` device and try to refactor the code to be DRY, and as a bonus you'll get missing `IMAGE_SIZE` variable for free, something like this:

```
define Device/dlink_dir-500
  IMAGE_SIZE := 7800k
  DEVICE_PACKAGES := jboot-tools kmod-usb2 kmod-usb-core \
                kmod-ledtrig-gpio kmod-ledtrig-netdev kmod-ledtrig-timer
  DLINK_FIRMWARE_SIZE := 0x7E0000
  KERNEL := $(KERNEL_DTB)
  IMAGES += factory.bin
  IMAGE/sysupgrade.bin := mkdlinkfw | pad-rootfs | append-metadata
  IMAGE/factory.bin := mkdlinkfw | pad-rootfs | mkdlinkfw-factory
endef

define Device/dlink_dir-506l
  $(Device/dlink_dir-500)
  DTS := DIR-506L
  DEVICE_TITLE := D-Link DIR-506L
  DLINK_ROM_ID := DLK6E2114001
  DLINK_FAMILY_MEMBER := 0x6E21
endef

define Device/dwr-512-b
  $(Device/dlink_dir-500)
  DTS := DWR-512-B
  DEVICE_TITLE := D-Link DWR-512 B
  DEVICE_PACKAGES += kmod-usb-serial kmod-usb-serial-option kmod-usb-net \
                kmod-usb-net-cdc-ether comgt-ncm
  DLINK_ROM_ID := DLK6E2412001
  DLINK_FAMILY_MEMBER := 0x6E24
endef
```